ui/core/src/assets.sifchain.sandpit.json

```diff
@@ -1,46 +1,447 @@
 {
   "assets": [
     {
-      "symbol": "cdtoken",
+      "symbol": "rowan",
       "decimals": 18,
-      "imageUrl": "https://assets.coingecko.com/coins/images/279/large/ethereum.png",
-      "name": "cdtoken",
-      "network": "sifchain"
+      "name": "Rowan",
+      "imageUrl": "./images/siflogo.png",
+      "address": "0xA3D31ee81Ec2a898B4CF7A67a0851086e4Da7af3",
+      "network": "ethereum"
+    },
+    {
+      "address": "0x66Cf623418Dba9f79c347d81C75e754DDaF788eF",
```

Comment:
These are off. Should be our sifchain wrapped tokens: `c<Token`

Reply:
Not sure what you mean - they look right to me.  For example,

```
    {
      "address": "0x2198f2bC87a9c754E19E44D0B87Fce06c0754C73",
      "decimals": 18,
      "imageUrl": "https://assets.coingecko.com/coins/images/1102/thumb/enjin-coin-logo.png?1547035078",
      "name": "Enjin Coin",
      "network": "ethereum",
      "priceUsd": 0.367884,
      "symbol": "cENJ",
      "transferLimit": "13591240717182590517248"
    },

```

The script could remove the address field, but does it matter?

Reply:
You're right. The network should be `sifchain` and the symbol should be all lowercase. Yea, no worries about the address field.